samlsp/middleware.go

```diff
@@ -300,6 +322,16 @@ func (m *Middleware) GetAuthorizationToken(r *http.Request) *AuthorizationToken
 		return nil
 	}
 
+	//decompress and decode attributes
+	attrList := tokenClaims.Attributes[jwtTokenClaims]
+	if len(attrList) == 1 {
+		gz, _ := gzip.NewReader(bytes.NewReader([]byte(attrList[0])))
```

Comment:
we're going to want error handling here

Reply:
Everything is generated internally here.  The gzip reader shouldn't fail creation with the in memory reader.